modules/minimp3/SCsub

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
+Import("env")
+Import("env_modules")
+
+env_minimp3 = env_modules.Clone()
+
+# Godot's own source files
+env_minimp3.add_source_files(env.modules_sources, "*.cpp")
+env_minimp3.disable_warnings()
```

Comment:
The warnings should only be disabled for thirdparty code, not for Godot files.
Here the thirdparty code is only headers so it's a bit tricky. I'd suggest trying without disabling warnings, and if some warnings are raised in the headers, then we'll have to use `-isystem` to include them to the include path.

Reply:
Okay, I wasn't sure about that one. I changed it to use `-isystem` now.